src/config/ConfigMetadata.ts

```diff
@@ -202,6 +202,18 @@ export const ConfigMetadata: {[key: string]: { type: string, defaultValue?: any
         type: 'boolean',
         defaultValue: false,
     },
+    [ConfigNames.THUNDRA_TRACE_INTEGRATIONS_AWS_SES_BODY_MASK]: {
```

Comment:
I don't think we need to have two distinct configurations for masking body and subject separately. For simpler configuration we can have a single config `THUNDRA_TRACE_INTEGRATIONS_AWS_SES_MAIL_MASK`